.github/workflows/rust-release.yml

```
@@ -295,6 +295,14 @@ jobs:
           # ${{ matrix.target }}
           dest="dist/${{ matrix.target }}"
 
+          # We want to ship the raw Windows executables in the GitHub Release
+          # in addition to the compressed archives. Keep the originals for
+          # Windows targets; remove them elsewhere to keep artifacts small.
+          keep_originals=false
```

I think this description is slightly more accurate:

```suggestion
          # Windows targets; remove them elsewhere to limit the number of
          # artifacts that end up in the GitHub Release.
          keep_originals=false
```